source/connect/connection-targets.txt

```diff
@@ -0,0 +1,150 @@
+.. _scala-connection-targets:
+
+==========================
+Choose a Connection Target
+==========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use a connection string and a ``MongoClient`` object
+to connect to different types of MongoDB deployments.
+
+Atlas
+-----
+
+To connect to a MongoDB deployment on Atlas, include the following elements
+in your connection string:
+
+- The URL of your Atlas cluster
+- Your MongoDB username
+- Your MongoDB password
+
+Then, pass your connection string to the ``MongoClient`` constructor.
+
+.. tip::
+
+   Follow the :atlas:`Atlas driver connection guide </driver-connection>`
+   to retrieve your connection string.
+
+When you connect to Atlas, we recommend using the {+stable-api+} client option to avoid
+breaking changes when Atlas upgrades to a new version of {+mdb-server+}.
+To learn more about the {+stable-api+} feature, see the :ref:`<scala-stable-api>`
+guide.
+
+The following code shows how to use the {+driver-short+} to connect to an Atlas cluster. The
+code also uses the ``serverApi()`` method to specify a {+stable-api+} version.
+
+.. literalinclude:: /includes/connect/connection-target.scala
+   :language: scala
+   :start-after: start-connect
+   :end-before: end-connect
+   :dedent:
+
+Local Deployments
+-----------------
+
+You can connect to a local MongoDB deployment in the following ways:
+
+- Instantiate a ``MongoClient`` object without any parameters to
+  connect to a MongoDB server running on ``localhost`` on port ``27017``:
+
+  .. literalinclude:: /includes/connect/connection-targets-local-replica.scala
+   :language: scala
+   :start-after: start-connect-local
+   :end-before: end-connect-local
+   :dedent:
+
+- Explicitly specify the ``hostname`` to connect to a MongoDB
+  instance running on the specified host on port ``27017``: 
+
+  .. literalinclude:: /includes/connect/connection-targets-local-replica.scala
+   :language: scala
+   :start-after: start-connect-local-host
+   :end-before: end-connect-local-host
+   :dedent:  
+
+- Explicitly specify the ``hostname`` and the ``port``:
+
+  .. literalinclude:: /includes/connect/connection-targets-local-replica.scala
+   :language: scala
+   :start-after: start-connect-local-host-port
+   :end-before: end-connect-local-host-port
+   :dedent:
+
+Replica Sets
```

Comment:
Have we decided in general not to show a sharded cluster example?

Reply:
@jyemin The team hasn't included information on connecting to a sharded cluster in any of the other drivers, but we can look into whether this makes sense. Thanks for the suggestion!